### PR TITLE
Add alr sbp

### PR DESCRIPTION
**Pull request by mortonjt**
Please complete the following checklist:

* [x] I have read the guidelines in [CONTRIBUTING.md](https://github.com/biocore/scikit-bio/blob/master/CONTRIBUTING.md).

* [x] I have documented all public-facing changes in [CHANGELOG.md](https://github.com/biocore/scikit-bio/blob/master/CHANGELOG.md).

* [x] **This pull request includes code, documentation, or other content derived from external source(s).** If this is the case, ensure the external source's license is compatible with scikit-bio's [license](https://github.com/biocore/scikit-bio/blob/master/COPYING.txt). Include the license in the `licenses` directory and add a comment in the code giving proper attribution. Ensure any other requirements set forth by the license and/or author are satisfied. **It is your responsibility to disclose code, documentation, or other content derived from external source(s).** If you have questions about whether something can be included in the project or how to give proper attribution, include those questions in your pull request and a reviewer will assist you.

* [ ] **This pull request does not include code, documentation, or other content derived from external source(s).**

**Note:** [REVIEWING.md](https://github.com/biocore/scikit-bio/blob/master/REVIEWING.md) may also be helpful to see some of the things code reviewers will be verifying when reviewing your pull request.

### CODE REVIEW

File: skbio/stats/tests/test_composition.py [hunk 1]
```diff
@@ -393,6 +394,104 @@ def test_ilr_inv_basis_one_dimension_error(self):
...
+                                 np.log(comp1[:, 2]/comp1[:, 1])]).T
+        alr2d_method = alr(comp1, denominator_idx=1)
+        B = 1/(1 + np.exp(alr2d_byhand[:, 0]) + np.exp(alr2d_byhand[:, 1]))
+        A = B * np.exp(alr2d_byhand[:, 0])
+        C = B * np.exp(alr2d_byhand[:, 1])
+        alrinv2d_byhand = np.column_stack((A, B, C))
+        alrinv2d_method = alr_inv(alr2d_method, denominator_idx=1)
+        npt.assert_allclose(alrinv2d_byhand, alrinv2d_method)
+
+        # 2d-composition
```
Comment (RNAer) [inline skbio/stats/tests/test_composition.py]: this is 1d, right? confusing comments. could u fix?

File: skbio/stats/composition.py [hunk 2]
```diff
@@ -642,6 +645,137 @@ def ilr_inv(mat, basis=None, check=True):
...
+    if len(mat.shape) == 2:
+        mat_idx = np.insert(mat, denominator_idx,
+                            np.repeat(0, mat.shape[0]), axis=1)
+        comp = np.zeros(mat_idx.shape)
+        comp[:, denominator_idx] = 1 / (np.exp(mat).sum(axis=1) + 1)
+        numerator_idx = list(range(0, comp.shape[1]))
+        del numerator_idx[denominator_idx]
+        for i in numerator_idx:
+            comp[:, i] = comp[:, denominator_idx] * np.exp(mat_idx[:, i])
+    elif len(mat.shape) == 1:
```
Comment (RNAer) [inline skbio/stats/composition.py]: similarly, you can simply the conditions to `.ndim` like previously.

**Comment by mortonjt** [comment]
Issue raised and comments addressed.  Thank you @RNAer !

**Comment by mortonjt** [comment]
Thanks!

**Pull request merged**